### PR TITLE
Add Kinova Lite arm support and possibility to launch from different computers 

### PR DESCRIPTION
**Pull request by KmakD**
I've also updated workflow to match other repositories and introduce versioning.
Can't however run workflow dispatch before merging so we have to pray the image will build :whale: . Or fix it later :upside_down_face:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,76 +99,95 @@ If neither `RB` nor `RT` is pressed, the robot operates in _regular_ driving mod
...
-1. Access NUC via ssh
+| ROS distro | Supported architectures |
+| :--------: | :---------------------: |
+|  `noetic`  |      `linux/amd64`      |
+
+Available on [Docker Hub](https://hub.docker.com/r/husarion/kortex-joystick/tags)
+
+### Demo
+
+#### Running on Panthers User Computer
```
Comment (macstepien) [inline README.md]:
```suggestion
#### Running on Panther's User Computer
```

File: README.md [hunk 2]
```diff
@@ -99,76 +99,95 @@ If neither `RB` nor `RT` is pressed, the robot operates in _regular_ driving mod
...
 
    ```bash
-   cd ~/husarion_ws/src/
    git clone https://github.com/husarion/kortex_joystick_demo.git
    ```
 
-3. Setup a gamepad according to [Setup joy](#setup-joy)
-
-4. Setup virtual desktop
+3. If you are using Kinova Gen3 Lite, edit [compose.kinova.yaml](./demo/compose.kinova.yaml) file and add `arm:=gen3_lite` launch parameter to the command
```
Comment (macstepien) [inline README.md]: Maybe instead we can use `.env` file, like it is used in other repositories, e.g. [rosbot-xl-manipualtion](https://github.com/husarion/rosbot-xl-manipulation/blob/main/.env) and define version there

File: README.md [hunk 3]
```diff
@@ -99,76 +99,95 @@ If neither `RB` nor `RT` is pressed, the robot operates in _regular_ driving mod
...
 
-   ```bash
-   cd kortex_joystick_demo/
-   source ./kortex_joystick_demo/scripts/setup_virtual_desktop.sh
+   ```yaml
+   command: >
+      roslaunch kortex_joystick_demo kortex_joystick_demo.launch
+        robot_ip_address:=10.15.20.4
```
Comment (macstepien) [inline README.md]: I think that also `robot_ip_address` can be in `.env` - or maybe `net.env` and put all IP configurations there

File: demo/compose.kinova.yaml [hunk 4]
```diff
@@ -1,13 +1,13 @@
 services:
 
   kortex-ros:
-    image: husarion/kortex-joystick:noetic
+    image: husarion/kortex-joystick:nightly
```
Comment (macstepien) [inline demo/compose.kinova.yaml]: after building new image it would be nice to change it to some exact version

**Review by macstepien — CHANGES_REQUESTED** [review]

File: demo/compose.kinova.yaml (hunk 4 above)
Comment (KmakD) [reply demo/compose.kinova.yaml]: Yes, but this will be another PR I guess

File: demo/compose.rviz.yaml [hunk 5]
```diff
@@ -14,8 +18,6 @@ services:
     environment:
       - DISPLAY=${DISPLAY:?Source of setup_virtual_desktop.sh required!}
       - LIBGL_ALWAYS_SOFTWARE=1
```
Comment (macstepien) [inline demo/compose.rviz.yaml]: to make it more standard this envs can be moved to `cpu-config` like [here](https://github.com/husarion/rosbot-xl-autonomy/blob/main/compose.pc.yaml)

**Review by macstepien — CHANGES_REQUESTED** [review]
Also please remember to change image: `image: kinova_test` after build.

**Review by macstepien — APPROVED** [review]

**Pull request merged**